sdk/spring/azure-spring-boot-test-aad/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
 # Release History
 
 ## 1.0.0 (Unreleased)
+### Breaking Changes
```

Comment:
There's no need to update the CHANGELOG.md for azure-spring-boot-test-xxx projects.